public/app/core/components/sidemenu/BottomSection.tsx

```diff
@@ -1,30 +1,85 @@
-import React from 'react';
-import { cloneDeep, find } from 'lodash';
+import React, { useState } from 'react';
+import { cloneDeep } from 'lodash';
+import { NavModelItem } from '@grafana/data';
+import { Icon, IconName } from '@grafana/ui';
+import appEvents from '../../app_events';
 import { SignIn } from './SignIn';
-import BottomNavLinks from './BottomNavLinks';
+import SideMenuItem from './SideMenuItem';
+import { ShowModalReactEvent } from '../../../types/events';
 import { contextSrv } from 'app/core/services/context_srv';
+import { OrgSwitcher } from '../OrgSwitcher';
+import { getFooterLinks } from '../Footer/Footer';
+import { HelpModal } from '../help/HelpModal';
 import config from '../../config';
-import { NavModelItem } from '@grafana/data';
 
 export default function BottomSection() {
   const navTree: NavModelItem[] = cloneDeep(config.bootData.navTree);
-  const bottomNav: NavModelItem[] = navTree.filter((item) => item.hideFromMenu);
```

Comment:
🤔 I am no sure about this change, is there any reason to remove the Interface TS checking?

Reply:
it just doesn't need it. because we define the type of `navTree` above it, when doing `navTree.filter` typescript can infer the type automatically 🙌